examples/relevant-words-example/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Yoast Prominent Words Analysis</title>
+    <link rel='stylesheet' id='source-sans-pro-css'
+          href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600&amp;subset=latin'/>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="../../dist/yoast-seo.min.css">
+
+    <script type="text/javascript" src="relevant-words-example-browserified.js"></script>
+</head>
+<body class="purple seo">
+
```

Comment:
I'd remove these whitespaces and indent the div, so it is clear it is a child of the body.